src/Native/System.Net.Http.Native/pal_easy.h

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+
+#include <curl/curl.h>
+
+enum
+{
+    CurlOptionLongBase = 0,
+    CurlOptionObjectPointBase = 10000,
+    CurlOptionFunctionPointBase = 20000,
+};
+
+enum PAL_CURLoption : int32_t
+{
+    PAL_CURLOPT_INFILESIZE = CurlOptionLongBase + 14,
```

Comment:
A naïve question: is there some advantage to using static_assert instead of using the macros from curl.h? Would that involve a cast?